Cargo.toml

```
@@ -15,9 +16,13 @@ edition = "2021"
 [lib]
 name = "dwt_systick_monotonic"
 
+[features]
+extend = []
+
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 cortex-m = "0.7"
```

```suggestion
cortex-m = "0.7.4"
```
When `DWT::cycle_count` got introduced and `DWT::get_cycle_count` [got deprecated](https://github.com/rust-embedded/cortex-m/pull/349).

Seems cortex-m intend to [keep `get_cycle_count` until `v0.8`](https://github.com/rust-embedded/cortex-m/pull/349#issuecomment-898885908) but since this is breaking anyway maybe just getting the latest is the best way forward? :thinking: 

(copied from other PR)

---

Absolutely! Thanks.